<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlottingDialog4</class>
 <widget class="QDialog" name="PlottingDialog4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>曲线样式管理</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog { background-color: white; color: black; font-family: &quot;Microsoft YaHei&quot;; }
QLabel { color: black; }
QLineEdit, QComboBox { border: 1px solid #555; padding: 2px; border-radius: 3px; color: black; background: white;}
QPushButton { border: 1px solid #555; padding: 4px; border-radius: 3px; background: #f0f0f0; color: black; }
QGroupBox { border: 1px solid #999; margin-top: 10px; padding-top: 10px; border-radius: 3px; font-weight: bold; }
QGroupBox::title { subcontrol-origin: margin; left: 10px; padding: 0 3px; }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupCurve1">
     <property name="title">
      <string>主曲线设置 (压力/压差)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_L1">
        <property name="text">
         <string>图例名称:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="lineLegend1"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_S1">
        <property name="text">
         <string>数据点形:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboShape1"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_C1">
        <property name="text">
         <string>点颜色:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnColor1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_LS1">
        <property name="text">
         <string>连接线型:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboLineStyle1"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_LC1">
        <property name="text">
         <string>线颜色:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btnLineColor1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_X1">
        <property name="text">
         <string>X轴列:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QComboBox" name="comboX1"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_Y1">
        <property name="text">
         <string>Y轴列:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QComboBox" name="comboY1"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupCurve2">
     <property name="title">
      <string>副曲线设置 (产量/导数)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_L2">
        <property name="text">
         <string>图例名称:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="lineLegend2"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_S2">
        <property name="text">
         <string>数据点形:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboShape2"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_C2">
        <property name="text">
         <string>点颜色:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnColor2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_LS2">
        <property name="text">
         <string>连接线型:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboLineStyle2"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_LC2">
        <property name="text">
         <string>线颜色:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btnLineColor2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PlottingDialog4</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PlottingDialog4</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>
